src/compiler/GF/Compile/PGFtoJSON.hs

```diff
@@ -0,0 +1,134 @@
+module GF.Compile.PGFtoJSON (pgf2json) where
+
+import PGF(showCId)
+import PGF.Internal as M
```

Comment:
```suggestion
import qualified PGF.Internal as M
```

I not a fan of non-qualified imports -- it makes the code difficult to understand

Reply:
```suggestion
import qualified PGF.Internal as M
import PGF.Internal (CncCat(..), CncFun(..), ...) -- whatever you need
```